examples/feature_alignment_example/server.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+from pathlib import Path
+from typing import Any, Dict, List, Tuple
+
+import flwr as fl
+import pandas as pd
+from flwr.common.parameter import ndarrays_to_parameters
+from flwr.common.typing import Metrics, Parameters
+
+from examples.models.logistic_regression import LogisticRegression
+from examples.simple_metric_aggregation import metric_aggregation, normalize_metrics
+from fl4health.client_managers.poisson_sampling_manager import PoissonSamplingClientManager
+from fl4health.feature_alignment.tab_features_info_encoder import TabularFeaturesInfoEncoder
+from fl4health.server.tabular_feature_alignment_server import TabularFeatureAlignmentServer
+from fl4health.strategies.basic_fedavg import BasicFedAvg
+from fl4health.utils.config import load_config
+
+DATA_PATH = "examples/feature_alignment_example/mimic3d_hospital1.csv"
+CONFIG_PATH = "examples/feature_alignment_example/config.yaml"
+
+
+def get_initial_model_parameters(input_dim: int, output_dim: int) -> Parameters:
+    initial_model = LogisticRegression(input_dim, output_dim)
+    return ndarrays_to_parameters([val.cpu().numpy() for _, val in initial_model.state_dict().items()])
+
+
+def fit_metrics_aggregation_fn(all_client_metrics: List[Tuple[int, Metrics]]) -> Metrics:
```

Comment:
Both of these are now reusable functions in `examples/simple_metric_aggregation.py` unless these functions differ from the ones implemented there. You can just import them from that file.